tests/docker_ssi/test_docker_ssi.py

```diff
@@ -157,15 +157,26 @@ def setup_injection_metadata(self):
         self._setup_all()
 
     @features.ssi_injection_metadata
-    @missing_feature(
-        context.library in ("python", "nodejs", "dotnet", "java", "php", "ruby"), reason="Not implemented yet"
+    @missing_feature(context.library in ("dotnet", "java", "php", "ruby", "nodejs"), reason="Not implemented yet")
```

Comment:
@robertomonteromiguel : I guess "dotnet", "java", "php", "ruby", "nodejs" and "python" the only tracers supporting SSI? 

@betterengineering  if no, we must remove the `[python]` in PR title, to test other ones.

Reply:
That's correct - we support six languages for SSI